src/state/opamSysInteract.ml

```diff
@@ -1163,6 +1220,7 @@ let update ?(env=OpamVariable.Map.empty) config =
     | Macports -> Some (`AsAdmin "port", ["sync"])
     | Msys2 -> Some (`AsUser (Commands.msys2 config), ["-Sy"])
     | Netbsd -> None
+    | Nix -> None
```

Comment:
What about:
```suggestion
    | Nix -> Some (`AsUser "nix-channel", ["--update"])
```

Reply:
This wouldn't work for flakes, but is perhaps better than nothing